LocalSettings.php

```diff
@@ -974,8 +974,7 @@
 		'default' => false,
 	],
 	'wgSharedDB' => [
-		'default' => 'metawiki', // REMOVE?
-		'test3wiki' => null,
+		'default' => null,
```

Comment:
If this is done and this has been tested through proxing from test3, that it was unnecessary, then this can probably be merged.

Reply:
Once it is completely removed (no reason to just set to null), this can probably be merged, if no issues on test3 I see no reason why there would be in production.